chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/surveillance/SurveillanceType.java

```diff
@@ -35,6 +36,9 @@ public SurveillanceType() {
     @XmlElement(required = true)
     private String name;
 
+    public SurveillanceType() {
```

Comment:
Any reason not to add the `@NoArgsConstructor` instead?

Reply:
Yeah, I had it there but when the DEV plan ran the schemagen step failed. Removing NoArgsConstructor and putting the actual constructor in the code was the only thing I could find that fixed the schemagen error. I do not know why but am open to other suggestions if anyone else knows. It was an error about non-static inner classes (which this class is not).

Reply:
There is a short version of that information in my commit message

Reply:
Oh, I remember something about that in the past related to schemagen. I'm fine with it as is then